src/rollup/index.ts

```diff
@@ -61,6 +61,7 @@ export interface InputOptions {
 	};
 
 	acorn?: {};
+	acornInjects?: Function[];
```

Comment:
How about just calling this `acornPlugins`?

Reply:
I'm happy to change the name as you see fit. I chose `acornInjects` to avoid the confusion between `acorn.plugins` (where plugins are enabled, e.g. `asyncIteration: true`) and `acornPlugins` (where plugin functions would be passed).

Reply:
Hmm perhaps `acornInjectPlugins` to be clearer then?

Reply:
I renamed this field to `acornInjectPlugins`. I can also rename it to `acronPlugins` if you prefer; perhaps it's not really all that confusing. I'll leave the final decision to you. Thanks.